services/reproducible-central/reproducible-central.tester.js

```diff
@@ -0,0 +1,54 @@
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('reproducible gav')
+  .get('/org.apache.maven/maven-core/3.9.9')
+  .expectBadge({
+    label: 'Reproducible Builds',
+    message: '47/47',
+    color: 'green',
+  })
```

Comment:
If you look at the services build or run your service tests locally using `npm run test:services -- --only="ReproducibleCentral"` you will see they are all failing. The first problem is that you're requesting the badges in SVG format, but then making an assertion about the response as if it were JSON. You need to request the URLs with a .json extension to make assertions about them like this. I've not looked into it any further than that.

Reply:
ok i did not know how to run the tests: now that you told me how to do, i can work seriously on them

Reply:
fixed: now automated unit tests work, thanks for your help